sentry-core/src/main/java/io/sentry/transport/AlwaysAllowedTransportGate.java

```diff
@@ -0,0 +1,12 @@
+package io.sentry.transport;
+
+/**
+ * A dummy implementation of the {@link ITransportGate} interface that always allows sending the
+ * events.
+ */
+public class AlwaysAllowedTransportGate implements ITransportGate {
```

Comment:
would it be possible to not be public, but only visible within the package? not only for this class, but for most of them

Reply:
Well, this does not need to be here at all actually. It's just a dummy impl to illustrate the usage. I will remove it.